packages/design-system-react-native/CHANGELOG.md

```diff
@@ -7,6 +7,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.4.0]
+
+### Added
+
+- Added attach money icon ([#823](https://github.com/MetaMask/metamask-design-system/pull/823))
+
+### Fixed
+
+- Jazzicon, Blockies and Maskicon icon generation for CAIP-10 addresses ([#816](https://github.com/MetaMask/metamask-design-system/pull/816))
+
 ## [0.3.1]
```

Comment:
Updating changelog to be more human readable and only including user facing PRs: New icon, CAIP 10 fixes